packages/components/src/SquiggleChart.tsx

```diff
@@ -26,7 +26,11 @@ export interface SquiggleChartProps {
   /** If the result is a function, where the function ends */
   diagramStop? : number,
   /** If the result is a function, how many points along the function it samples */
-  diagramCount? : number
+  diagramCount? : number,
+  /** variables declared before this expression */
+  environment? : exportEnv,
+  /** When the environment changes */
+  onEnvChange?(env: exportEnv): void
```

Comment:
linty nitpick? do you want spaces here? (but like I said, we won't talk about it)

Reply:
Haha, possibly? I might run it through prettier and see what it things.

Reply:
nah let me worry about dx/devops this week and see how far i get building out the PR flow

Reply:
prettier doesn't care either way